### ISSUE TITLE
Add release purls for CHEBI

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/purl.obolibrary.org#147]
See https://github.com/ebi-chebi/ChEBI/issues/3135

Releases here:
ftp://ftp.ebi.ac.uk/pub/databases/chebi/archive/

redirect URLs would be of the form:

```
ftp://ftp.ebi.ac.uk/pub/databases/chebi/archive/rel100/ontology/chebi.{obo,owl}
```

One problem is that the CHEBI OWL is not the same as the OBO OWL - https://github.com/ebi-chebi/ChEBI/issues/3105

So it may be better to have our own archive derived from the EBI one

**Comment by mcourtot** [comment]
ChEBI is migrating their OWL towards OBO OWL as per [ebi-chebi/ChEBI#3105](https://github.com/ebi-chebi/ChEBI/issues/3105#issuecomment-179236461) . It's probably better to rely on their conversion (and help/support them as needed) than create another archive.

**Comment by cmungall** [comment]
ok, we now use chebi's OWL (the linked ticket above is fixed)

we need a redirect

http://purl.obolibrary.org/obo/chebi/161/chebi.owl

from

ftp://ftp.ebi.ac.uk/pub/databases/chebi/archive/rel161/ontology/chebi.owl

for any value of N

I think we can use any apache redirect rule in the yml @jamesaoverton @kltm ?

**Comment by balhoff** [comment]
I fixed this a while back in #649.

**Comment by matentzn** [comment]
All done, thanks @balhoff
